src/wstool/multiproject_cli.py

```diff
@@ -840,6 +843,20 @@ def cmd_set(self, target_path, argv, config=None):
                 "Config path does not match %s %s " % (config.get_base_path(),
                                                        target_path))
 
+        if options.unmanaged:
+            outputs2 = multiproject_cmd.cmd_find_unmanaged_repos(config)
+            table2 = get_info_table(config.get_base_path(),
+                                   outputs2,
+                                   data_only=True,
+                                   unmanaged=True)
+            for line in table2.splitlines():
+                if options.confirm:
+                    line += " -y"
+                if options.do_update:
+                    line += " -u"
```

Comment:
BTW: you think auto-updating makes sense here? We added it more for the use-case that someone adds a new element to an empty location. In the case of scraping, that would seem possible (add to wstool, then run update), but odd.